features/_your_results/period_percent_table.feature

```diff
@@ -0,0 +1,16 @@
+Feature:
+  As a user who has just completed the WPCC tool, I should be able to see helpful information on how the yearly increments is done.
+
+  Scenario: Display the minimum contribution percents information table
+    Given I am on the Your Details step
+    And I complete the your details form and move to your contributions page
+    And I fill in my contributions:
+        | your_contribution | employer_contribution |
+        | 1                 | 1                     |
+
+    And I move on to the results page
+    Then I should see a link to the legal minimum contributions table
+    And I should see a table with a column heading for each period
+    And I should see in the percents information table:
```

Comment:
You should hopefully be able to simplify this Scenario a bit further.
```
    Given I have valid details
    And I have valid contributions
    And I am on the Your Results step
```
would be the most basic form to get you onto the results page. 
If you want to be explicit about what contributions you are making then you can leave the `And I fill in my contributions:` step.

I also think that these two steps could be combined into one..? wdyt?
```
And I should see a table with a column heading for each period
And I should see in the percents information table:
```